custom-elements/parser/parser-uses-constructed-element.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Custom Elements: Changes to the HTML parser</title>
```

Comment:
Not sure why the thing being tested is hidden in `<meta name="assert">`, while multiple test files have the same `<title>`... I'd prefer to use `<title>` for this. But metadata nits do not really matter.